firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -6534,6 +6534,11 @@ extension String {
         tableName: nil,
         value: nil,
         comment: "Tile title for Twitter")
+    public static let DefaultSuggestedX = MZLocalizedString(
+        key: "SuggestedSites.X",
```

Comment:
"X" seemed a bit vague for a key, but I can change it to that if we want to match the others?